### ISSUE TITLE
Buttons are read as unavailable without button label when changing page

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#287]
If you press space on a button; for example first page, last page, previous page or next button, when a new page is loaded and button is no longer available the only thing read is unavailable. This could be misunderstood by the customers, because you are not told what is unavailable, but it could also be that the users understands that it is the button that is no longer available. We need to check this with A360.

**Comment by haxxmaxx** [comment]
As long as this is accompanied with a confirmation that the page has changed, it is no problem that the name of the button is not announced

**Comment by mlb76** [comment]
This was not to be fixxed.  After talking to Casey we said that no fix was needed for this issue.
